interface/usergroup/user_admin.php

```diff
@@ -356,6 +356,9 @@ function authorized_clicked() {
     <?php echo generate_select_list("erxrole", "newcrop_erx_role", $iter['newcrop_user_role'],'',xl('Select Role'),'','','',array('style'=>'width:150px')); ?>
 </td>
 </tr>
+<tr>
+<td><span class="text"><?php echo xlt('Weno Provider ID'); ?>: </span></td><td><input type="text" name="erxprid" style="width:150px;"  value="<?php echo $iter["weno_prov_id"]?>"></td>
```

Comment:
revisiting these edit requests with the new software. Interesting things it's doing. So much to learn.